File: client/Reduce/impl/schema/ShortInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Reduce.impl.schema
{
    class ShortInfo
    {

        private Int32 id;

        public Int32 Id
        {
            get { return id; }
            set { id = value; }
        }

        private Int32 userId;

        public Int32 UserId
        {
            get { return userId; }
            set { userId = value; }
        }

        private String shortUrl;

        public String ShortUrl
        {
            get { return shortUrl; }
            set { shortUrl = value; }
        }

        private String url;

        public String Url
        {
            get { return url; }
            set { url = value; }
        
        }

        private Int32 frequency;

        public Int32 Frequency
        {
            get { return frequency; }
            set { frequency = value; }
        }

        private long createTime;

        public long CreateTime
        {
            get { return createTime; }
            set { createTime = value; }
        }


    }
}

File: service/src/main/java/org/coody/reduce/domain/ShortInfo.java
package org.coody.reduce.domain;

import java.util.Date;

import org.coody.framework.jdbc.annotation.DBVague;
import org.coody.framework.jdbc.entity.DBModel;

@SuppressWarnings("serial")
public class ShortInfo extends DBModel {

	private Long id;

	private Integer userId;

	@DBVague
	private String url;

	private Long frequency;

	private Integer appId;

	/**
	 * 1正常 2已停用
	 */
	private Integer status;

	private Date createTime;

	public Integer getAppId() {
		return appId;
	}

	public void setAppId(Integer appId) {
		this.appId = appId;
	}

	public Integer getStatus() {
		return status;
	}

	public void setStatus(Integer status) {
		this.status = status;
	}

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public Integer getUserId() {
		return userId;
	}

	public void setUserId(Integer userId) {
		this.userId = userId;
	}

	public String getUrl() {
		return url;
	}

	public void setUrl(String url) {
		this.url = url;
	}

	public Date getCreateTime() {
		return createTime;
	}

	public void setCreateTime(Date createTime) {
		this.createTime = createTime;
	}

	public Long getFrequency() {
		return frequency;
	}

	public void setFrequency(Long frequency) {
		this.frequency = frequency;
	}

}
